python/audio_dsp/dsp/biquad.py

```diff
@@ -331,6 +313,13 @@ class biquad_slew(biquad):
     ----------
     slew_shift : int
         The shift value used in the exponential slew.
+    b_shift : int
+        The number of right shift bits applied to the b coefficients.
+        The default coefficient scaling allows for a maximum coefficient
+        value of 2, but high gain shelf and peaking filters can have
+        coefficients above this value. Shifting the b coefficients down
+        allows coefficients greater than 2, with the cost of b_shift
+        bits of precision.
```

Comment:
I think the b_shift is an attribute rather than a parameter. We might be able to inherit it from the base class